Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -39,9 +39,7 @@ def teardown
   end
 
   def needs_test_cmd_taps
-    unless ENV["HOMEBREW_TEST_OFFICIAL_CMD_TAPS"]
-      skip "HOMEBREW_TEST_OFFICIAL_CMD_TAPS is not set"
-    end
+    skip "HOMEBREW_TEST_OFFICIAL_CMD_TAPS is not set" unless ENV["HOMEBREW_TEST_OFFICIAL_CMD_TAPS"]
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.